docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -188,11 +188,150 @@ if (sdlManager.getSystemCapabilityManager().isCapabilitySupported(SystemCapabili
+
+@![iOS]
+##### Objective-C
+```objc
+SDLGetSystemCapability *remoteControl = [[SDLGetSystemCapability alloc] initWithType: SDLSystemCapabilityTypeRemoteControl];
+[self.sdlManager sendRequest:remoteControl withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if (!response.success) { return; }
+    SDLRemoteControlCapabilities *remoteCapabilities = ((SDLGetSystemCapabilityResponse *)response).systemCapability.remoteControlCapability;
+    <#Save Remote Capabilities#>
+}];
+```
+##### Swift
+```swift
+sdlManager.systemCapabilityManager.subscribe(toCapabilityType: .seatLocation, with: { capability in
+    guard let seats = capability.seatLocationCapability?.seats else { return }
+    <#Save Remote Capabilities#>
+})
+```
+!@
+
+With the saved remote capabilities struct you can build a UI/UX to display modules to the user. When the user selects the module you can send remote control RPCs using the `moduleInfo`s `moduleId` property. 
+
+!!! Note
+`ModuleID` is only required for Core v6.0+.  Examples will be shown later in the documentation on how to use `moduleID`.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+NSString *selectedModuleID = <#SelectedModule#>.moduleInfo.moduleID;
+```
+
+##### Swift
+```swift
+let selectedModuleID = <#SelectedModule#>.moduleInfo?.moduleId
+```
+!@
+
+### Get Consent
+This feature is only available for Core v6.0+. Some OEMs may wish to ask the driver for consent before a user can control a module. This is typically done automatically by the HMI depending on the configuration. However, the `SDLGetInteriorVehicleDataConsent` RPC will alert the driver for consent in some OEMs if the module if not free "being used by another user"  and `allowMultipleAccess` "multiple users can access/set the data at the same time" is true. `allowMultipleAccess` is part of the `moduleInfo` in the module object.
+
+Check the `allowed` property in the `SDLGetInteriorVehicleDataConsentResponse` to see what modules can be controlled. Note the order of the `allowed` array is 1-1 with the `moduleIDs` array you passed into the `SDLGetInteriorVehicleDataConsent` RPC.
+
+!!! Note
+You should always try to get consent before setting any module data. If consent is not granted you should not attempt to set any module's data.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+[self.sdlManager sendRequest:[[SDLGetInteriorVehicleDataConsent alloc] initWithModuleType:@"<#ModuleType#>" moduleIds:@[@"<#ModuleID#>",@"<#ModuleID#>",@"<#ModuleID#>"]] withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
```

Comment:
Create a variable of the RPC, don't pass it directly into `sendRequest`. Please fit with other documentation.

Reply:
fixed!